config/profiles/latency.yaml

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: (c) Copyright 2023 Advanced Micro Devices, Inc.
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: onload-latency-profile
+data:
+  # SPDX-License-Identifier: MIT
```

Comment:
Onload profiles have a different license: BSD-2-Clause.

Shall we also add more headers to specify the source file and the version, e.g. https://github.com/Xilinx-CNS/onload/blob/master/src/driver/linux_onload/cplane.c#L1715.